runtime/doc/pattern.txt

```diff
@@ -131,7 +131,7 @@ gD			Goto global Declaration.  When the cursor is on a
 
 							*CTRL-C*
 CTRL-C			Interrupt current (search) command.  Use CTRL-Break on
-			MS-DOS |dos-CTRL-Break|.
+			Windows |dos-CTRL-Break|.
```

Comment:
I'm not too sure about this one too. I'm fairly certain you can interrupt with `CTRL-C` on Windows...